packages/core/src/dialects/abstract/query-generator.js

```diff
@@ -1172,6 +1172,7 @@ export class AbstractQueryGenerator extends AbstractQueryGeneratorTypeScript {
   selectQuery(tableName, options, model) {
     options = options || {};
     const limit = options.limit;
+    const shouldMinifyAlias = options.minifyAliases ?? this.options.minifyAliases;
```

Comment:
setting the default option should actually be done one step higher: in queryInterface (⚠️ there could be more than one queryInterface method that uses selectQuery).

Doing so removes the need to check the global option in `postgres/query.js`

Reply:
Hey @ephys I just realized something we have this [Unit test](https://github.com/sequelize/sequelize/blob/main/packages/core/test/unit/query-generator/select-query.test.ts#L510)  however the check what we are doing now relies on the `query-interface` which is causing this test to fail. Integration tests should pass as expected. So for that now instead of relying on the queryInterface's alias minification I am passing directly! Is that correct?

Reply:
You can give the option to `selectQuery` directly in that test, there is no need for a custom sequelize instance anymore